ui/pages/first-time-flow/create-password/new-account/new-account.component.js

```diff
@@ -99,11 +102,8 @@ export default class NewAccount extends PureComponent {
 
       this.context.trackEvent({
         category: EVENT.CATEGORIES.ONBOARDING,
-        event: 'Submit Password',
-        properties: {
-          action: 'Create password',
-          legacy_event: true,
-        },
+        event: EVENT_NAMES.ACCOUNT_PASSWORD_CREATED,
+        properties: {},
```

Comment:
wondering, would it be okay to omit properties if it's empty?

Reply:
i believe an error is triggered if category, event or properties fields are omitted.  i like being explicit here that no properties are being set on this event.